sky/authentication.py

```diff
@@ -221,6 +221,10 @@ def setup_gcp_authentication(config):
                         f'but the file {config_path} does not contain the '
                         'account information.')
         config['auth']['ssh_user'] = account.replace('@', '_').replace('.', '_')
+        
+        # Generating ssh key if it does not exist
+        _, public_key = get_or_generate_keys(private_key_path, public_key_path)
```

Comment:
We can remove the return value here, since the key content will not be used below. Probably, just:
```
# Generating ssh key if it does not exist
get_or_generate_keys(private_key_path, public_key_path)
```

Reply:
OK, I removed return value